cfgov/cfgov/urls.py

```diff
@@ -394,17 +394,17 @@ def flagged_wagtail_template_view(flag_name, template_name):
                 r'^search/',
                 include('search.urls')),
 
-    flagged_url('TDP_RELEASE',
+    flagged_url('TDP_CRTOOL',
                 r'^practitioner-resources/youth-financial-education/curriculum-review/tool/',  # noqa: E501
                 include_if_app_enabled('teachers_digital_platform',
                                        'teachers_digital_platform.tool-urls')),
 
-    flagged_url('TDP_RELEASE',
+    flagged_url('TDP_CRTOOL',
             r'^practitioner-resources/youth-financial-education/curriculum-review/before-you-begin/',  # noqa: E501
             include_if_app_enabled('teachers_digital_platform',
                                     'teachers_digital_platform.begin-urls')),
 
-    flagged_url('TDP_RELEASE',
+    flagged_url('TDP_CRTOOL_PROTOTYPES',
```

Comment:
Final question: Do you want to go ahead and set up the flagged URL for the search interface, so you don't have to make a PR for that later?

Reply:
That is a great question.  Yes, yes I do want to go ahead and create that now :-)... just a moment